Neo4j.Driver/Neo4j.Driver/Internal/AsyncSession.cs

```diff
@@ -84,6 +85,11 @@ bool reactive
             LastBookmarks = Bookmarks.From(config.Bookmarks);
             _initialBookmarks = LastBookmarks;
         }
+
+        if (config.NotificationsConfig != null)

```

Comment:
Could use null-coalescing operator here. Although changes the style from existing lines. Just a thought, no real preference from me.

Reply:
the null check was pointless so removed.